modules/product/src/VariationFieldInjection.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+namespace Drupal\commerce_product;
+
+use Drupal\commerce_product\Entity\ProductVariationInterface;
+use Drupal\Core\Ajax\AjaxResponse;
+use Drupal\Core\Ajax\ReplaceCommand;
+use Drupal\Core\Entity\EntityFieldManagerInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Core\Field\BaseFieldDefinition;
+
+/**
+ * Default implementation of VariationFieldInjectionInterface.
```

Comment:
I've started ommitting the docblock in these cases, cause "Default implementation of X" says absolutely nothing.